packages/compass-e2e-tests/helpers/compass.js

```diff
@@ -233,15 +236,18 @@ async function startCompass(
 
 /**
  * @param {string[]} logs The main process console logs
- * @returns {Promise<any[]>}
+ * @returns {Promise<{raw: Buffer, structured: any[]}>}
  */
 async function getCompassLog(logs) {
   const logOutputIndicatorMatch = logs
     .map((line) => line.match(/Writing log output to (?<filename>.+)$/))
     .find((match) => match);
   if (!logOutputIndicatorMatch) {
     debug('no log output indicator found!');
-    return [];
+    return {
+      raw: null,
```

Comment:
Let’s be consistent with the more precise typings above :)

```suggestion
      raw: Buffer.alloc(0),
```